platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/build_services.adoc

```diff
@@ -184,8 +171,9 @@ When this property has no value, which is the default, Gradle does not constrain
 When this property has a value > 0, Gradle will allow no more than the specified number of tasks to use the service concurrently.
 
 IMPORTANT: When the consuming task property is annotated with `@Internal`, for the constraint to take effect, the build service *must* be registered with the consuming task via
-link:{groovyDslPath}/org.gradle.api.Task.html#org.gradle.api.Task:usesService(org.gradle.api.provider.Provider)[Task.usesService(Provider<? extends BuildService<?>>)].
-This is not necessary if, instead, the consuming property is annotated with `@ServiceReference`.
+link:{groovyDslPath}/org.gradle.api.Task.html#org.gradle.api.Task:usesService(org.gradle.api.provider.Provider)[Task.usesService(Provider<? extends BuildService<?>>)]. NOTE: at this time, Gradle cannot discover indirect usage of services (for instance, if an additional service is used only by a service that the task uses directly).
```

Comment:
```suggestion
link:{groovyDslPath}/org.gradle.api.Task.html#org.gradle.api.Task:usesService(org.gradle.api.provider.Provider)[Task.usesService(Provider<? extends BuildService<?>>)]. 
At this time, Gradle cannot discover indirect service usage (for instance, if an additional service is used only by a service that the task uses directly).
```

Reply:
I tried something like that, but this would break formatting - lines 174/175 would no longer be part of the "IMPORTANT" block, and they are additions related to the content in that block. I considered having two IMPORTANT blocks, one after the other, and it felt weird to me. 

Any alternative suggestions, @lkasso?

Reply:
Two solutions:
1. Feel free to keep it on the same line but just change it to:  `NOTE: at this` -> `Note that at this`
2. You can also use a `+` to make the formating work. `NOTE: at this` -> 
```
Task.usesService(Provider<? extends BuildService<?>>)].  
+
Note that at this time,
```